backend/api/v1beta1/api-reference-docs/index.html

```diff
@@ -0,0 +1,7400 @@
+<!DOCTYPE html>
```

Comment:
Why is this a new file but not a diff to an existing file? (actually I think there should be almost no diff to v1 doc?)
Maybe we the generate files are only to be moved over into kubeflow/website repo?

Reply:
That's right. These are new files. The v1 api reference docs [changed a little](https://github.com/kubeflow/website/pull/3444/files#diff-bd577926284caf918471395eb589421ee226402fde7bcb2dd026743c5846cdad). I think this is a good opportunity to sync the v1 reference docs, as we are rolling out v2 api.

The old workflow is cumbersome because we store SOT in a different repo. So if someone changes API, it's not apparent if the changes need to be reflected on kubeflow.org. Now, a diff will signal that we need to send a separate PR updating the docs. Besides, it may be convenient for some people to have reference documentation close to the proto files.